index.html

```diff
@@ -46,30 +48,256 @@ <h2 id="jobTitle">I am a Software Developer</h2>
+                <path
+                    d="M152.591 151.842C150.652 152.379 148.645 151.244 148.108 149.307C147.571 147.369 148.707 145.363 150.646 144.825C152.585 144.288 154.592 145.423 155.129 147.361C155.666 149.298 154.53 151.305 152.591 151.842Z"
+                    fill="white" stroke="#262626" stroke-width="0.496227" stroke-miterlimit="10" />
+                <path d="M79.7207 237.489L79.7188 248.712L68.8173 257.512L68.9336 246.702L79.7207 237.489Z" fill="white"
+                    stroke="#262626" stroke-width="0.496227" stroke-miterlimit="10" />
+                <path d="M119.26 254.197L119.351 272.173L102.444 282.971L102.525 265.614L119.26 254.197Z" fill="white"
+                    stroke="#262626" stroke-width="0.496227" stroke-miterlimit="10" />
             </g>
             <defs>
-            <clipPath id="clip0_48_297">
-            <rect width="406.889" height="312.031" fill="white" transform="matrix(-0.963665 0.267112 0.267112 0.963665 392.104 0)"/>
-            </clipPath>
+                <clipPath id="clip0_48_297">
+                    <rect width="406.889" height="312.031" fill="white"
+                        transform="matrix(-0.963665 0.267112 0.267112 0.963665 392.104 0)" />
+                </clipPath>
             </defs>
-            </svg>
-            
-        </main>
+        </svg>
+    </main>
+    <!-- Project/works Section -->
+    <section class="work-section-grid">
+        <div class="recent-work grid-content">
+            <div id="topic">
+                <p>My Recent Works</p>
+                <hr>
+            </div>
+            <img id="img_01" src="images\img_01.png" alt="">
+            <div id="project-row1-RHS">
+                <h2 id="story">Multi-Post Stories</h2>
+                <p id="story-text">A daily selection of privately personalized
+                    reads; no accounts or sign-ups required. has
+                    been the industry's standard dummy text ever
+                    since the 1500s, when an unknown printer took
+                    a standard dummy text.</p>
+                <div class="project-link">
+                    <p class="css">css</p>
+                    <p class="html">html</p>
+                    <p class="bootstrap">bootstrap</p>
+                    <p class="ruby">Ruby</p>
+                </div>
+                <div><a class="goto_project">See Project</a></div>
+            </div>
+        </div>
+
+        <div class="content_1 grid-content"><img src="images\img_4.png" alt="">
+            <h3 class="title-post">Profesional Art Printing Data More</h3>
+            <p class="supporting-text">A daily selection of privately personalized
+                reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <div class="project-link">
+                <p class="html">html</p>
+                <p class="bootstrap">bootstrap</p>
+                <p class="ruby">Ruby</p>
+            </div>
```

Comment:
- [ ] Please make sure to use a `ul` with the necessary `li` items inside here instead of single `p` tags. This is to keep your code a bit more semantic.